src/applications/simple-forms/20-10207/pages/nonVeteranPhoneAndEmail.js

```diff
@@ -30,30 +29,31 @@ let pageSchema = {
   },
 };
 
-// test on dev before making this change
-if (environment.isDev() || environment.isLocalhost()) {
-  pageSchema = {
-    uiSchema: {
-      ...titleUI(({ formData }) => getPhoneAndEmailPageTitle(formData)),
-      nonVeteranPhone: phoneUI('Phone number'),
-      nonVeteranEmailAddress: emailToSendNotificationsUI({
-        // no living situation risk = has housing = has email
-        required: formData => formData.livingSituation?.NONE,
-        updateUiSchema: formData => ({
-          'ui:options': {
-            hint: getPhoneAndEmailPageEmailHint(formData),
-          },
-        }),
+/** @type {PageSchema} */
```

Comment:
code should be same here - just refactored all these so the diff will be clean when i remove above code